com.unity.testing.urp/Scripts/Runtime/CustomRenderPipeline/CustomRenderer.cs

```diff
@@ -4,14 +4,21 @@ namespace UnityEngine.Rendering.Universal
 {
     public class CustomRenderer : ScriptableRenderer
```

Comment:
Does this test make sense?
With this change we're now using internal functionality not available to users.

Reply:
That is good question, I personally would prefer if we had this test as separate project. Because not using this interal function, will result in lots of variants that we do not need

Reply:
ohhh that is a good point. Then I think we should just do what you've done here actually.